app/models/spree/promotion_decorator.rb

```diff
@@ -0,0 +1,4 @@
+Spree::Promotion.class_eval do
+  belongs_to :store
```

Comment:
Was following the existing structure of the other models, but we might be able to